domain/src/main/kotlin/com/neki/domain/map/infra/persist/BrandRepositoryAdapter.kt

```
@@ -13,8 +15,27 @@ import org.springframework.stereotype.Repository
  */
 @Repository
 class BrandRepositoryAdapter(private val jpaRepository: JpaBrandRepository) : BrandRepository {
+    override fun save(brand: Brand): Brand {
+        TODO("Not yet implemented")
```

detekt (1.23.8)

[warning] The NotImplementedDeclaration should only be used when a method stub is necessary. This defers the development of the functionality of this function. Hence, the `NotImplementedDeclaration` should only serve as a temporary declaration. Before releasing, this type of declaration should be removed.

(detekt.exceptions.NotImplementedDeclaration)